piwebapi_samples/Data Analysis/README.md

```diff
@@ -0,0 +1,53 @@
+
+We will be utilizing the data that we have and do some basic data analysis and predictive machine learning using jupyter notebook in order to predict the cooling time for different units.
+
+### Data Overview
+In this sample, we are denoting these control systems as two different `VAVCO` units which are represented as AF Elements. Each of these elements have different attributes which mimic an actual cooling unit such as `% cooling` and `Set Point Offset`.
+
+There are also eventframes of the template `VAVCO startup` which represent the start up events of the cooling unit. They have some attributes which capture the metadata on start time as well as end time. 
+
+All this data is available as part of `Building Data.xml`. There is a helpful utility which will import this database as well as create and populate the PI Points. Find more on how to use it in the next section.
+
+## Getting Started
+- Clone the GitHub repository
+- Open the Data Analysis folder with your IDE
+- Install the required modules by running the following command in the terminal : `pip install -r requirements.txt
+### Setting up database and PI Archive
+- In the Data Analysis folder, populate the values of `test_config.json` with your own system configuration. 
+For example:
+```
+	"PIWEBAPI_URL": "https://mydomain.com/piwebapi/",
+	"AF_SERVER_NAME": "AssetServerName",
+	"PI_SERVER_NAME": "PIServerName",
+	"AF_DATABASE_NAME": "AFDatabaseNAme",
+	"USER_NAME": "MyUserName",
+	"USER_PASSWORD": "MyUserPassword",
+	"AUTH_TYPE": "basic" # Basic or Kerberos 
+```
+
+- Run `PIUploadUtility.sln` which imports the database from `Building Example.xml`, creates PI tags outlined in `tagdefinition.csv` and uploads the values in `pidata.csv` to PI Data Archive.
+- You can open up PI System Explorer and check out the database you just created to further understand the data and the hierarchy.
+
+### Running Jupyter Notebook
+- Open a terminal and type in `jupyter notebook`. This will open a browser window. Navigate to the cloned repository and open up `Sandbox.ipynb`. Run the cells one by one and you can see the output in browser itself.
+- The last cell in the notebook is for running unit tests so that you can test your PI Web API connection is working properly. As it tests the methods defined earlier in the notebook, you need to run the previous cells (the one which defines the GET and POST methods) of the notebook before trying to run the unit tests.
+
+##  Documentation
+The documentation for the various controllers and topics can be found at your local PI Web API help system: https://your-server/piwebapi/help
+
+##  Authentication and minimum versions
+The sample works with Basic and Kerberos authentication methods. 
+This sample has been tested on `PI Web API 2018 SP1`, `PI AF Server 2018 SP2` and `PI Data Archive 2018 SP2`.
```

Comment:
End of the file usually has a few links, see some other READMEs for examples.

Reply:
Will add